UnityProject/Assets/Scripts/Objects/Pipes/MonoPipe.cs

```diff
@@ -178,6 +181,73 @@ public void SetColour(Color newColour)
 			Colour = newColour;
 		}
 
+		#region Vent Crawl
+
+		protected void DoVentCrawl(HandApply interaction, GasMix pipeMix)
+		{
+			if (pipeData.ConnectedPipes.Count == 0)
+			{
+				Chat.AddExamineMsgFromServer(interaction.Performer, $"There is no pipe connected to this {gameObject.ExpensiveName()} theres no reason to enter");
```

Comment:
Might still be useful to quickly hide? Just let 'em know they can't move.
```suggestion
				Chat.AddExamineMsgFromServer(interaction.Performer, $"There is no pipe connected to this {gameObject.ExpensiveName()}, there's no reason to enter");
```
Ditto line `209`